docs/release/osg-23.md

```diff
@@ -0,0 +1,152 @@
+    -   Adds an option to allow running `condor_ce_trace` without a SciToken for testing batch system integration
+
+-   [XRootD 5.6.2](https://github.com/xrootd/xrootd/blob/v5.6.2/docs/ReleaseNotes.txt#L1-L123):
+    an update from XRootD 5.5.1 in OSG 3.6 main.
+    -  New Features
+        -   Add xrdfs cache subcommand to allow for cache evictions
+        -   Better handling of unicode strings in the API
+        -   Add gsi option to display DN when it differs from entity name
+        -   Allow specfication of minimum and maximum creation mode
+        -   Make maxfd be configurable (default is 256k)
+        -   Include token information in the monitoring stream (phase 1)
+        -   Implement a file evict function
+        -   Increase default number of parallel event loops to 10
+        -   xrdcp: number of parallel copy jobs increased from 4 to 128
+        -   Allow XRootD to return trailers indicating failure
+        -   Denote Accept-Ranges in HEAD response
+        -   Report cache object age for caching proxy mode
+        -   Allow origin to be a directory of a locally mounted file system
+        -   Implement ability to have the token username as a separate claim
+        -   Use SHA-256 for signatures, and message digest algorithm
+        -   Allow option '-tokenlib none' to disable token validation
+        -   Allow to point to a token file using CGI `'?xrd.ztn=tokenfile'`
+    -   Major bug fixes
+        -   Fix SEGV in case request has object for opaque data but no content
+        -   Fix memory leaks in GSI authentication
+        -   Fix chunked PUT creating empty files
+
+-   [GlideinWMS 3.10.5](http://glideinwms.fnal.gov/doc.v3_10_5/history.html):
+    an update from 3.10.1 in OSG 3.6 main
+    
+    !!! warning "If you are using custom setup scripts..."
+        If you are using custom setup scripts please change the use of `glidein_config`:
+
+        -   Custom scripts should always read values via `gconfig_get()`.
+            The only exception is the parsing of the line to get the `add_config_line` source file.
+        -   `add_config_line` is deprecated in favor of `gconfig_add`.
+            `add_config_line` will be removed from future versions.
+        -   `add_config_line_safe` is deprecated in favor of `gconfig_add_safe`.
+            `gconfig_add` is the recommended method to use also in concurrent scripts.
+        -   Custom scripts in Python should `import gconfig.py` (compatible with both Python 2 and 3) and use the
+            provided functions or classes: `gconfig_get`, `gconfig_add`, etc.
+
+    -   This release completes EL9 and Python 3.9 support
+    -   Added structured logging
+    -   Various OSG_Autoconf improvements
+    -   Fixed bugs with Python 3.9 and rrdtools failures with missing ClassAds and monitoring
+
+#### Package Removals ####
+
+The following packages were removed from OSG 23 compared to OSG 3.6:
+
+-  `blahp`: available as `condor-blahp`
```

Comment:
No more condor-blahp package. blahp components are in the main condor package.

Reply:
Fixed